site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,90 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supported storage addons attached to workloads: you need to run 
+`copilot svc deploy` in order to deploy the storage, and the storage is deleted along with the service
+when you run `copilot svc delete`.
+
+Now, you have the option to create environment-level storage addons: the storage is deployed when you run `copilot env deploy`,
+and isn't deleted until you delete the whole environment by running `copilot env delete`.
+
+Similar to the workload-level storage, the environment-level storage is, under the hood, just another [environment addon](../docs/developing/addons/environment.en.md)!
```

Comment:
Maybe consider removing it as we use "addons" everywhere?

Reply:
Yeah I think we can keep the link like "Now, you have the option to create [environment-level storage addons](../docs/developing/addons/environment.en.md): the storage..."

Reply:
I do feel like keeping the link in this place flows better 🤔 🤔 😓 

> "Now, you have the option to create [environment-level storage addons](https://github.com/aws/copilot-cli/docs/developing/addons/environment.en.md): the storage..."

People will be disrupted to click into another link.

Whereas 

> Similar to the workload storage, the environment storage is, under the hood, just another [environment addon](http://0.0.0.0:8000/copilot-cli/docs/developing/addons/environment/)!

People will know to investigate what "environment addons" is after having a general/vague understanding of what "environment storage" is compared to "workload storage"

<img width="788" alt="Screenshot 2023-02-17 at 1 20 34 PM" src="https://user-images.githubusercontent.com/79273084/219767122-fed6adfe-7397-4361-8197-27c823306da6.png">